content/rc/how-to/aws-zone-mapping.md

```diff
@@ -1,84 +1,68 @@
-    AWS user](#unprivileged-user-creation).
-1. Press on **Map Zones** button.
+1. [Log in](https://app.redislabs.com/#/login) to your Redis Cloud Essentials account.
+1. Go to **settings** > **account**.
+1. Click **Map your AWS Zones**.
+1. Enter the Access Key ID and Secret Access Key for an [unprivileged AWS user](#unprivileged-user-creation).
+1. Click **Map Zones**.
 
-Allow the process a few minutes to complete as it maps the availability
-zones. Once complete, you see this:
+The zone mapping takes a few minutes to complete as it maps the availability zones.
+During the zone mapping you see:
 
 ![zone-mapping](/images/rc/zone-mapping.png?width=600&height=359)
 
-When the process is complete, the next time you go to create a new
-subscription, the drop down for "Cloud" has the zone letter in
-it.
+After zone mapping is complete, the dropdown for **Cloud** in a new subscription shows the zone letter.
 
 ![after_zone_mapping](/images/rc/after_zone_mapping.png?width=600&height=192)
 
-For existing subscriptions in Redis Cloud Essentials, you now see the Availability Zone
-they are using. To reduce the latency described above:
+For existing subscriptions in Redis Cloud Essentials, you now see the Availability Zone they are using.
+To reduce the latency described above:
 
-1. [Create a new
-    subscription]({{< relref "/rc/administration/setup/create-subscription.md" >}}) in
-    the correct AZ
-1. [Create a new
-    database]({{< relref "/rc/administration/setup/create-database.md" >}})
-1. [Migrate your
-    data]({{< relref "/rc/how-to/importing-dataset-redis-cloud.md" >}})
-    over to the new database.
+1. [Create a new subscription]({{< relref "/rc/administration/setup/create-subscription.md" >}}) in the AZ for your application.
+1. [Create a new database]({{< relref "/rc/administration/setup/create-database.md" >}}).
+1. [Migrate your data]({{< relref "/rc/how-to/importing-dataset-redis-cloud.md" >}}) to the new database.
 
-While simple, this causes some downtime. If downtime is not an
-option, [please contact
-support](mailto:support@redislabs.com?Subject=Zero%20Downtime%20DB%20Migration)
-for further assistance.
+While simple, this causes some downtime.
+To prevent any downtime during the zone mapping, contact [support](mailto:support@redislabs.com?Subject=Zero%20Downtime%20DB%20Migration) for further assistance.
 
 ## Unprivileged user creation
 
-Redis Cloud Essentials requires an AWS IAM user to perform the zone mapping. It is
-recommended to create a new and restricted account for security reasons.
-To do this:
+Redis Cloud Essentials requires an AWS IAM user for the zone mapping.
+We recommend that you create a new and restricted account to use for zone mapping.
 
-1. Create a new user in your AWS account **where your application is
-    hosted**.
+To create a restricted IAM account for zone mapping:
+
+1. Create a new user in your AWS account where your application is hosted.
```

Comment:
Same comment as above.  Should we provide links to the IAM doc for each of these tasks?  Must the user be an admin user or a delegated user? I"m guessing the later, which has been my experience when integrating cloud-based apps with IAM.